lib/standards.h

```diff
@@ -44,16 +44,27 @@ struct Standards {
     enum cppstd_t : std::uint8_t { CPP03, CPP11, CPP14, CPP17, CPP20, CPP23, CPP26, CPPLatest = CPP26 } cpp = CPPLatest;
 
     /** --std value given on command line */
-    std::string stdValue;
+    std::string stdValueC;
+
+    /** --std value given on command line */
+    std::string stdValueCPP;
 
     bool setC(std::string str) {
-        stdValue = str;
```

Comment:
I just wonder if `setC` and `setCPP` are needed.

If only setStd is called from outside we could implement all the logic in that directly or make setC and setCPP private? I feel maybe the code would be a bit simpler but on the other hand testability is important too.